src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -180,7 +180,9 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
       Utils.withBusyState(setLoadingWorkspace)
     )(async () => {
       try {
-        const workspace = await Ajax(signal).Workspaces.workspace(namespace, name).details()
+        const workspace = await Ajax(signal).Workspaces.workspace(namespace, name).details([
+          'accessLevel', 'canCompute', 'canShare', 'owners', 'workspace', 'workspace.attributes', 'workspaceSubmissionStats'
```

Comment:
This needs to include `workspace.authorizationDomain` (verify by trying to copy a notebook to another workspace)

Reply:
Good catch, missed that one, the only needing to specify sub-keys that are objects thing can be tricky.